### PR TITLE
[FIX] website: correct wrong query count after httpocalypse

### PR DESCRIPTION
**Pull request by rdeodoo**
An error regarding tests was not catch during httpocalypse [1] review.
The test query counts were lowered in website by 2 everywhere, but it shouldn't
have as there were no SQL perf improvement from this refactoring regarding
website.
The query count actually decreased because the test-only SQL Savepoints got
divided by 2, probably thanks to a low level test improvement in the PR.
But outside of tests, there were no SQL Query gained.
This should have been reflected in the `EXTRA_REQUEST` variable instead.

Before:
Savepoint > Rollback > Savepoint > [Actual SQL Queries] > Rollback
After:
Savepoint > [Actual SQL Queries] > Rollback

It is important to fix it because:
1. (Minor) There is a de-sync between the query count in the test and the
   actual query count (in the logs outside of tests), making it hard to figure.
2. (Major) Some controller got then wrongly lowered to 0 query counts instead
   of 2. An incoming PR in master would then make those tests expects negative
   query counts, which doesn't make any sense.

[1]: https://github.com/odoo/odoo/pull/78857
Note that the exact commit can't be found by bisect as those don't work on
their own, DB can't be started due to circular import.

### CODE REVIEW

**Review by xmo-odoo — APPROVED** [review]

File: addons/website/tests/test_performance.py [hunk 1]
```diff
@@ -3,16 +3,13 @@
 
 from odoo.tests.common import HttpCase
 
-EXTRA_REQUEST = 4 - 1
+EXTRA_REQUEST = 2 - 1
 """ During tests, the query on 'base_registry_signaling, base_cache_signaling'
-won't be executed on hot state, but new queries related to the test cursor
-will be added::
+won't be executed on hot state, but new queries related to the test cursor will
+be added:
```
Comment (xmo-odoo) [inline addons/website/tests/test_performance.py:9]: This just creates unnecessary diff-churn.

**Review by Julien00859 — APPROVED** [review]
I was always bugged how the tests reported 0 query, it makes sense now! Thank you :)

File: addons/website/tests/test_performance.py (hunk 1 above)
Comment (rdeodoo) [reply addons/website/tests/test_performance.py:9]: Indeed, thank for mentioning.
Since I am editing that comment and the constant value, I wanted to fix that `::` typo (that actually comes from https://github.com/odoo/odoo/commit/17e6a69b9189127f2988d05850d2594244c2ff2c where the un-indent also seems like an useless diff in the first place AFAIK, even if I do agree it is better written without the indent level)

Since I am fixing that `::` part, I might as well rewrite both sentences to make them use the available 80 char space while I am touching it.

**Review by rdeodoo — COMMENTED** [review]
> I was always bugged how the tests reported 0 query, it makes sense now! Thank you :)

Well, to be fair we should have investigated that sooner as indeed it meant something was wrong somewhere. At the end I came with a new PR in master which brings the query count in the assert to `-1`. Cool but.. not possible (yet? :smile: )

**Comment by rdeodoo** [comment]
@robodoo r+

**Comment by rdeodoo** [comment]
@robodoo rebase-ff
